lib/main.mlx

```diff
@@ -179,38 +169,37 @@ let feedback () =
-  ; "Does it support Windows?", <p>"No, not yet. Watch this space for news!"</p>
-  ; "Can I build the Developer Preview of Dune from sources?",
+  ; "Is it stable?",
+    <p>
+      "No. The Developer nightly is built from unreleased code on the head of the development branch. If you use the nightly, expect bugs and please report them to "
+      <Link href="https://github.com/ocaml/dune/issues">"ocaml/dune"</Link>
+      ". For a stable release of OCaml, please follow the installation guide on "
+      <Link href="https://ocaml.org">"OCaml.org"</Link>
+      ". Binary installations of stable release are available at"
+      <Link href="https://github.com/ocaml-dune/dune-bin-install?tab=readme-ov-file#dune-binary-installer">"Dune Binary Installer"</Link>"."
+    </p>
+  ; "Can I install the binary on Windows?", <p>"No, not yet. Watch this space for news!"</p>
+  ; "Can I build the nightly of Dune from sources?",
     <div class_="flex flex-col gap-2">
-      <p>"Yes. The only difference from a regular build from source is in a few
-      configuration flags. Please look in the Dune repository for complete instructions
-      on how to do a source build of Dune."
+      <p>"Yes. The nightly is just a binary distribution of the latest Dune features."</p>
+      <p>
+        "Check out "
+	<Link href="https://github.com/ocaml/dune?tab=readme-ov-file#installation-1">"the installation instructions in the Dune repository"</Link>
+	" on how to do a source build of Dune."
       </p>
-      <p>"When configuring the build you'll want to enable the following flags:" </p>
-      <Code>
-"--pkg-build-progress enable"
-"--lock-dev-tool enable"
-"--portable-lock-dir enable"
-</Code>
     </div>
   ; "Can I access these features from a version of Dune managed by opam?",
     <div class_="flex flex-col gap-2">
-      <p>"Yes, but we can't guarantee they will work correctly since the Dune Developer Preview makes some assumptions for package management that aren't supported by opam."
+      <p>"Yes, but we can't guarantee that every feature of the nightly will make it as-is into future releases."
       </p>
-      <p>"To enable these features make sure you are pinning dune to the development version, and export the following environment variables: "
+      <p>"To enable these features make sure you are pinning Dune to the development version: "
       </p>
       <Code>
-"$ opam pin add dune --dev"
-"$ export DUNE_CONFIG__PKG_BUILD_PROGRESS=enabled"
-"$ export DUNE_CONFIG__LOCK_DEV_TOOL=enabled"
-"$ export DUNE_CONFIG__PORTABLE_LOCK_DIRS=enabled"
+[JSX.string "$ opam pin add dune --dev"]
```

Comment:
I think the full flag is

```suggestion
[JSX.string "$ opam pin add dune --dev-repo"]
```

Maybe its best to use the full flag rather than the shortened version?